documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md

```diff
@@ -49,6 +51,16 @@ The `routes` option allows you to customise the [`_routes.json`](https://develop
 
 You can have up to 100 `include` and `exclude` rules combined. Generally you can omit the `routes` options, but if (for example) your `<prerendered>` paths exceed that limit, you may find it helpful to manually create an `exclude` list that includes `'/articles/*'` instead of the auto-generated `['/articles/foo', '/articles/bar', '/articles/baz', ...]`.
 
+### Generate 404
+
+The `generate404` option allows you to specify whether a fallback `404.html` page will be generated.
```

Comment:
I wonder if we should rename this to `generateFallback` so it better matches with the error message users get during build.
```bash
> Using @sveltejs/adapter-cloudflare
file:///Users/my-app/node_modules/@sveltejs/kit/src/core/postbuild/fallback.js:53
        throw new Error(`Could not create a fallback page — failed with status ${response.status}`);
```